### PR TITLE
Sak-30577  - Web Content Fixes

### PR DESCRIPTION
**Pull request by AndrewWJordan**
SAK-30577 - Changed layout of web content edit form and changed placement of edit buttons. Could not replicate other ui issues in ticket. 

Hopefully this pull request is correct. Please let me know what would need to be done to correct it if it is not.  Thank you.

### CODE REVIEW

**Comment by juanjmerono** [comment]
Could you squash these two commits in one?, following this guide:
https://confluence.sakaiproject.org/display/SAKDEV/Git+Setup

File: reference/library/src/morpheus-master/sass/modules/tool/_base.scss [hunk 1]
```diff
@@ -59,8 +59,7 @@
 		float: right;
 		display: block;
 		position: relative;
-		margin: 0px 0 -20px 0;
-		top: 9px;
+		margin: -1em 0 0 0;
```
Comment (SedueRey) [inline reference/library/src/morpheus-master/sass/modules/tool/_base.scss]: Could you add this modification to the tool itself? in `sass/modules/tool/iframe` under the namespace `#{$namespace}sakai-iframe`

This class is used all along sakai and could be dangerous to modify for all.

Comment (AndrewWJordan) [inline reference/library/src/morpheus-master/sass/modules/tool/_base.scss]: I squashed those commits and updated the branch SAK-30577.  I can also make @SedueRey suggested tool mod.

Comment (SedueRey) [inline reference/library/src/morpheus-master/sass/modules/tool/_base.scss]: Yes, please, could you do that update and then squash your commits again? Thanks :)

**Comment by SedueRey** [comment]
Looks great! Could you squash the two commits in just one?

**Comment by juanjmerono** [comment]
Wow !! The Squash and Merge option madethe squash for you Andrew !!

**Comment by AndrewWJordan** [comment]
Amazing.. Was going to squash and commit today, but if you're all set then we shall venture forth! Is there another Morpheus issue you two would recommend I tackle next?

**Comment by SedueRey** [comment]
@AndrewWJordan Do you think you can handle this? https://jira.sakaiproject.org/browse/SAK-29494

**Comment by AndrewWJordan** [comment]
This sounds good. Would love to help with this. I'll get started asap.

**Comment by AndrewWJordan** [comment]
Thanks guys!  That was a bit rocky, hopefully the next round will be smoother!

**Pull request merged**